package_schemas/country_estimates/2_country_estimates_22.json

```diff
@@ -0,0 +1,356 @@
+{
+    "scheming_version": 1,
+    "dataset_type": "country-estimates-22",
+    "name": "Country Estimates 2022",
+    "link_name": "Country Estimates 2022",
+    "about": "",
+    "about_url": "http://github.com/ckan/ckanext-scheming",
+    "add_data_on_create": false,
+    "dataset_fields": [{
+            "field_name": "title",
+            "label": "Title (autogenerated):",
+            "form_snippet": "text.html",
+            "validators": "autogenerate",
+            "classes": ["hidden"],
+            "template": "{} Inputs UNAIDS Estimates 2022",
```

Comment:
shouldn't we get read of "UNADIS Estimates" from files & variables names all together?

Reply:
Yeah excellent catch.  Thankyou.